tests/test_validator_integration.py

```
@@ -0,0 +1,915 @@
+        }
+
+        not_forwarded = []
+        for field in schema_training_fields:
+            if field in handled_elsewhere:
+                continue
+            kwarg = schema_to_kwarg.get(field, field)
+            if kwarg not in forwarded:
+                not_forwarded.append(field)
+
+        # These are the remaining documented gaps (dead config fields)
+        expected_gaps = {
```

Ruff (0.15.1)

[error] Local variable `expected_gaps` is assigned to but never used

Remove assignment to unused variable `expected_gaps`

(F841)